custom_components/waste_collection_schedule/waste_collection_schedule/source/pronatura_bydgoszcz.py

```diff
@@ -0,0 +1,138 @@
+import datetime
+import json
+import logging
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Bydgoszcz Garbage Collection"
+DESCRIPTION = "Source for Bydgoszcz city garbage collection by Pronatura"
+URL = "https://zs5cv4ng75.execute-api.eu-central-1.amazonaws.com/prod"
```

Comment:
URL is used by the updatedoculinks.py script and will be shown in the README.md use the website URL here instead and rename this variable to something like API_URL